src/Compilers/Core/Portable/Compilation/IOperation.cs

```diff
@@ -104,6 +104,10 @@ public enum OperationKind
         // Newly added
 
         ConditionalAccessExpression,
-        IncrementExpression
+        IncrementExpression,
+
+        Argument,
```

Comment:
Apparently I'm delusional. Apparently either I blurred old VB semantics with VB.Net, or I'm just wrong, but there are no true omitted arguments in VB. I'll give Gen's suggestion some thought.